inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -62,13 +62,13 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::AlgebraicSimplification>(); // may introduce fake dynamism
     manager.register_pass<ngraph::pass::ConstantFolding>();
     manager.register_pass<ngraph::pass::ConvertScatterElementsToScatter>(); // partially depends on CF
-    manager.register_pass<ngraph::pass::DepthToSpaceFusion>();
+    manager.register_pass<ngraph::pass::DepthToSpaceFusion, false>();
```

Comment:
No, this transformations are registered in a right place in the pipeline of transformations. And some of them are required only in one or two plugins. So instead of disabling this transformation in every plugin we just enable them in plugins where they are really needed.